pkgs/development/libraries/zlib/default.nix

```diff
@@ -21,7 +22,7 @@ assert shared || static;
 
 assert splitStaticOutput -> static;
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
This needs
 - https://github.com/NixOS/nixpkgs/pull/214340
 
 (couldn't push to your branch)